js/wizard.js

```diff
@@ -0,0 +1,437 @@
+function validateIPaddress(ipaddress) {
+  if (/^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.test(ipaddress)) {
+    return (true);
+  }
+  return (false);
+}
+
+function validateFQDN(fqdn)
+{
+  if (/^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/.test(fqdn)) {
+    return (true);
+  }  return (false);
+
+}
+
+function validateNetmask(netmask)
+{
+  if (/^(254|252|248|240|224|192|128)\.0\.0\.0|255\.(254|252|248|240|224|192|128|0)\.0\.0|255\.255\.(254|252|248|240|224|192|128|0)\.0|255\.255\.255\.(254|252|248|240|224|192|128|0)$/.test(netmask)) {
+    return (true);
+  }
+  return (false);
+}
+
+function convertNetmaskToCIDR(netmask) {
+  var maskNodes = netmask.match(/(\d+)/g);
+  var cidr = 0;
+  for(var i in maskNodes)
+  {
+    cidr += (((maskNodes[i] >>> 0).toString(2)).match(/1/g) || []).length;
+  }
+  return cidr;
+}
+
+
+function saveFile(filename, data) {
+  var blob = new Blob([data], {type: 'text/csv'});
+  if(window.navigator.msSaveOrOpenBlob) {
+    window.navigator.msSaveBlob(blob, filename);
+  }
+  else{
+    var elem = window.document.createElement('a');
+    elem.href = window.URL.createObjectURL(blob);
+    elem.download = filename;
+    document.body.appendChild(elem);
+    elem.click();
+    document.body.removeChild(elem);
+  }
+}
+
+$('#wifi-fields').hide();
+$('#wired-fields').hide();
+$('#dns-fields').hide();
+$('#ntp-fields').hide();
+$('#common_errors').hide();
+$('#wifi-no-dhcp-fields').hide();
+$('#wired-no-dhcp-fields').hide();
+$('input[name=wifidhcp]').prop('checked', 'checked');
+$('input[name=wireddhcp]').prop('checked', 'checked');
+$('input[id=screenlyv1]').prop('checked', 'checked');
+
+$('input[name=screenlyversion]').change(function() {
+  console.log("Screenly version changed");
+  var ntp = document.getElementById("ntp")
+  if($('input[id=screenlyv1]').is(':checked'))
+  {
+    console.log("Screenly v1 is selected");
+    ntp.disabled = false;
+  }
+  else
+  {
+    console.log("Screenly v2 is selected");
+    ntp.disabled = true;
+    $('#ntp-fields').hide();
+  }
+});
+
+$('input[name=wifi]').change(function() {
+  if($(this).is(':checked'))
```

Comment:
Since we are using Vanilla with jQuery wouldn't it become much more simple to use `if(this.checked)`?